test/envs/kubevirt.yaml

```diff
@@ -13,10 +13,10 @@ profiles:
         io.containerd.grpc.v1.cri:
           device_ownership_from_security_context: true
     network: $network
-    cpus: 4
-    memory: "8g"
+    cpus: 8
+    memory: "16g"
```

Comment:
same as the disk suggestion, maybe consider something less aggressive than doubling? 12?

Reply:
The windows server needs 8gs, and we have one cluster, so 16g is fine. This is the total memory for 3 clsuters with regional-dr.yaml.